### PR TITLE
Made a script to make creating the league config files easier

### PR DESCRIPTION
**Pull request by tanelso2**

### CODE REVIEW

**Comment by tanelso2** [comment]
Not exactly fully done, but I think it's far enough along to be usable.

File: scripts/make-league.js [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+    const response = await query("Is this ok? [y/n]");
+    switch (response) {
+        case 'y':
+        case 'Y':
+            return true;
+        case 'n':
+        case 'N':
+            return false;
+        default:
+            console.log(`I don't understand, try again`);
```
Comment (rcrobert) [inline scripts/make-league.js:31]: Pissmonger would suffer no fools while taking input, please fix this politeness.

File: scripts/make-league.js [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+    const name = await query('coach name');
+    const nickname = await query('nickname');
+    const id = await query('coach id');
+    const teamName = await query('team name');
+    const teamType = await query('team type');
+    const retVal = { name, id, teamName, teamType, nickname };
+    console.log(`This team looks like ${JSON.stringify(retVal)}`);
+    const ok = await okPrompt();
+    if (!ok) {
+        console.log(`Ok, let's try inputting that again`);
```
Comment (rcrobert) [inline scripts/make-league.js:81]: As above, far too kind. Ruins immersion.

File: scripts/make-league.js [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+async function editObjectInEditor(obj, toText, fromText) {
+    const text = toText(obj);
+    const randomName = `tmp-${Math.floor(Math.random() * Math.floor(1000))}`;
+    const tmpFile = path.join(os.tmpdir(), randomName);
+    // create file
+    // write text to file
+    await fs.writeFile(tmpFile, text);
+    openFileInEditor(tmpFile);
+    // read file
+    const result = await fs.readFile(tmpFile, {encoding: 'utf-8'});
```
Comment (rcrobert) [inline scripts/make-league.js:65]: Could clean the file here after editing is done?

**Review by rcrobert — APPROVED** [review]

File: scripts/make-league.js (hunk 3 above)
Comment (tanelso2) [reply scripts/make-league.js:65]: tmp dir gets cleared out on system reboot, so I think it's fine?

File: scripts/make-league.js (hunk 1 above)
Comment (tanelso2) [reply scripts/make-league.js]: Fixed

File: scripts/make-league.js (hunk 2 above)
Comment (tanelso2) [reply scripts/make-league.js]: Fixed

**Pull request merged**